cmd/prombench/Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN \
 
 COPY prombench /prombench/prombench
 COPY Makefile /prombench/Makefile
+COPY Makefile.common /prombench/Makefile.common
 COPY manifests/prombench /prombench/manifests/prombench
 
 WORKDIR /prombench
+ENTRYPOINT [ "make", "-C","/prombench/","deploy"] 
```

Comment:
The `ENTRYPOINT` instruction gets executed in `/github/workspace`.

Contents of the Dockerfile are present in `/prombench` dir.
Setting PROMBENCH_PATH is a good idea. We just need to pass this as an env var. I will test this